docs/design/referral-manager.md

```diff
@@ -0,0 +1,206 @@
+
+# Title: Referral Manager
+
+## Abstract
+
+Right now, referral program only works per satellite meaning a user on us-east-1 satellite can only refer people to join us-ease-1. 
+However, it's crucial for the tardigrade branded satellites to be able to handle cross-satellite referrals. Therefore, we plan to have a program running in GCP along with tardigrade satellites to handle this issue.
```

Comment:
It does not need to be GCP, correct? 
Let's call it party in the network instead of _program_